internal/validations/validations_os.go

```diff
@@ -0,0 +1,43 @@
+package validations
+
+import (
+	"bufio"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/openshift/check-payload/internal/types"
+)
+
+func ValidateOperatingSystem(mountPath string) types.OperatingSystemInfo {
+	info := types.OperatingSystemInfo{
+		Certified: false,
+		Error:     nil,
+	}
+
+	path := filepath.Join(mountPath, "/etc/redhat-release")
+	info.Path = path
+	f, err := os.Open(path)
+	if err != nil {
+		e := errors.New("release file not found")
+		info.Error = e
+		return info
+	}
+	defer f.Close()
+
+	certifiedReleases := []string{
+		"Red Hat Enterprise Linux release 9.2 (Plow)",
```

Comment:
We should move this to config so we can specify certified OS per release.
For 4.16 it will be 9.2 etc.

Reply:
Great idea.

Reply:
We expect to have multiple certified versions per OpenShift release, right?

Reply:
For 4.16, the base images will be 9.2 except for the node itself which will be on 9.4. So, yes, it will be good to make this a list to support multiple versions.